content/docs/setup/kubernetes/platform-setup/azure/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Platform setup for Azure
+title: Azure
 description: Instructions to setup the Azure cluster for Istio.
 weight: 15
```

Comment:
I recommend changing the weights so that the docs in this platform-setup section show up in alpha order